harness/tests/experiment/fixtures/pytorch_xor_model.py

```diff
@@ -405,3 +405,11 @@ def train_batch(
             self.context.step_optimizer(self.optimizer)
 
         return {"loss": loss}
+
+
+class XORTrialWithApexAmp(XORTrial):
```

Comment:
Let's avoid any new XOR-based tests.  Please use the linear `pytorch_onevar_model` instead.

The reason is: XOR-based tests only test that something runs; they're much harder to test if e.g. the gradients were updated as we would expect them to be.

Reply:
This is done.